Note one-way for private endpoints (#1721)
MicrosoftDocs/dataexplorer-docs: data-explorer/security-network-managed-private-endpoint-create.md

# Create a managed private endpoint for Azure Data Explorer

> Managed private endpoints are required to connect to Azure resources that are highly protected. They are one-way private connections for data flowing from Azure Data Explorer to other protected services. In this article, you'll learn how to create a managed private endpoint and connect it to your data source.

Reviewer: ```suggestion
Managed private endpoints are required to connect to Azure resources that are highly protected. They are one-way private connections which allow Azure Data Explorer to connect to other protected services. In this article, you'll learn how to create a managed private endpoint and connect it to your data source.
```
